ci/meta.yml

```diff
@@ -144,3 +144,117 @@ meta:
           dir: concourse-smoke-tests
         inputs:
           - name: terraform-output
+    generate-positive-proxy-test-manifest:
+      task: generate-positive-proxy-test-manifest
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((docker-awscli.repository))
+            version: ((docker-awscli.version))
+            tag: ((docker-awscli.version))
+        outputs:
+          - name: positive-proxy-test-manifest
+        run:
+          path: sh
+          dir: positive-proxy-test-manifest
+          args:
+          - -exc
+          - |
+            cat << EOF > manifest.json
+            {
+                "required_endpoints": {
+                  "GitHub":  "https://github.com",
+                }
+            }
+            EOF
+            cat manifest.json
+    generate-negative-proxy-test-manifest:
+      task: generate-negative-proxy-test-manifest
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((docker-awscli.repository))
+            version: ((docker-awscli.version))
+            tag: ((docker-awscli.version))
+        outputs:
+          - name: negative-proxy-test-manifest
+        run:
+          path: sh
+          dir: negative-proxy-test-manifest
+          args:
+          - -exc
+          - |
+            cat << EOF > manifest.json
+            {
+              "required_endpoints": {
+                "MSN": "https://msn.com"
+              }
+            }
+            EOF
+            cat manifest.json
+    run-positive-proxy-test:
```

Comment:
Same as above, these are the same, but with one input difference, so could be an env var. Example here -> https://github.ucds.io/dip/aws-ingestion/blob/eabff1fc670832de2f350fa1e584df6c3ddaf390/ci/load-test/meta-load-test.yml#L23